src/components/MapBase/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ViewerContainer } from '@datapunt/asc-ui';
+import { Zoom } from '@datapunt/amsterdam-react-maps/lib/components';
+import styled from '@datapunt/asc-core';
+import { Map as MapComponent, TileLayer } from '@datapunt/react-maps';
+
+const Wrapper = styled(MapComponent)`
+  height: 450px;
+  width: 100%;
+`;
```

Comment:
This should not be in this component; should be the responsibility of the implementing components